CONTRIBUTING.md

```diff
@@ -30,6 +30,20 @@ of the manual][8].  Adding information like the backtrace and any sample message
 and/or topic configurations used to the bug report makes it easier to track down bugs.
 Some steps to reproduce a bug reliably would also make a huge difference.
 
+## Sign-off on Commits
```

Comment:
In this context, I think "sign off" is a verb. There is no hyphen. The same is probably true for some of the comments below.